Sources/Exporters/OpenTelemetryProtocolHttp/logs/OtlpHttpLogExporter.swift

```diff
@@ -23,22 +24,28 @@ public class OtlpHttpLogExporter : OtlpHttpExporterBase, LogRecordExporter {
   }
   
   public func export(logRecords: [OpenTelemetrySdk.ReadableLogRecord], explicitTimeout: TimeInterval? = nil) -> OpenTelemetrySdk.ExportResult {
-    pendingLogRecords.append(contentsOf: logRecords)
-    let sendingLogRecords = pendingLogRecords
-    pendingLogRecords = []
-    
+  var sendingLogRecords: [ReadableLogRecord]!
+    dispatchQueue.sync {
+      pendingLogRecords.append(contentsOf: logRecords)
+      sendingLogRecords = pendingLogRecords
+      pendingLogRecords = []
+    }
```

Comment:
instead of this you can use:
```
var sendingLogRecords: [ReadableLogRecord] = []
exporterLock.withLockVoid {
    pendingLogRecords.append(contentsOf: logRecords)
    sendingLogRecords = pendingLogRecords
    pendingLogRecords = []      
  }
```